rosdep/python.yaml

```diff
@@ -701,6 +701,12 @@ python-falcon:
   fedora: [python-falcon]
   ubuntu: [python-falcon]
 python-fcn-pip:
+  debian:
+    pip:
+      packages: [fcn]
```

Comment:
thank you. I added depends. just to  make sure that these values are not rosdep key, but actual package name.